compute/README.md

```diff
@@ -0,0 +1,55 @@
+---
+page_type: sample
+languages:
+- golang
+products:
+- azure
+description: "These code samples will show you how to manage Compute using Azure SDK for Golang."
+urlFragment: compute
+---
+
+# Getting started - Managing Compute using Azure Golang SDK
+
+These code samples will show you how to manage Compute using Azure SDK for Golang.
+
+## Features
+
+This project framework provides examples for the following services:
+
+### Compute
+* Using the Azure SDK for Golang - Compute Management Library [compute/armcompute](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/compute/armcompute) for the [Azure Compute API](https://docs.microsoft.com/en-us/rest/api/compute/)
+
+### Prerequisites
+* an [Azure subscription](https://azure.microsoft.com)
+* Go 1.13 or above
+
+### Quickstart
+
+1. Clone the repository.
+
+    ```
+    git clone https://github.com/Azure-Samples/azure-sdk-for-go-samples.git
```

Comment:
Because we put this in the `new-version` branch and it is not a default branch for now, to reduce confusion of customers, we need to add the branch flags here to ensure if they run this command, it will automatically checkout to this branch
```suggestion
    git clone https://github.com/Azure-Samples/azure-sdk-for-go-samples.git --branch new-version
```